### ISSUE TITLE
[Feature Request] A toggle to enable CDH, even when SABnzbd is in a Paused state

### ISSUE THREAD

**Issue by vp-en** [mylar3/mylar3#1348]
**Describe the request**
A toggle to enable CDH even when SABnzbd is paused. I can imagine that some people might want it that way - but it would be nice to have a toggle to enable CDH even when SABnzbd is in a paused state.

Currently, mylar3 will not do any "imports" or "post-processing" (CDH/Completed Download Handling) when SABnzbd is in a paused state. This is a problem for me as I use the SABnzbds [quota] (https://sabnzbd.org/wiki/configuration/4.0/switches) system which pauses SABnzbd when it reaches the defined quota.

And then there can be a lot of already downloaded comics waiting in SABnzbds "history" that don't get any CDH treatment. This is a bit odd, as applications like Sonarr/Radarr and so on can still do imports as long as the download is actually complete. Having SABnzbds in a paused state should not, in my opinion, completely block any/all CDH :(.

**How did you come up with the idea?**
Radarr and Sonarr still do imports, even if SABnzbd is in a paused state.
Mylar3 gives this message in the logs: `SABnzbd has the active queue Paused. CDH will not work in this state.`

**Environment (please complete the following information):**
 - OS: Linux/Docker (LSIO)
 - Branch / commit: v0.7.1 (master)

**Additional information**
<3

**Comment by vp-en** [comment]
Also, slightly related question, does CDH remove completed/imported downloads from the SABnzbd history? (like Sonarr/Radarr does)

**Comment by evilhero** [comment]
Honestly, probably not going to happen anytime soon. That would involve changing the entire way CDH works against the history of SAB, which does not cycle thru past downloads. If it does happen it would be a low priority aspect as there are far more outstanding items that have a higher priority in the queue so to speak. 

CDH will only monitor the sabnzbd history of the client for the active download. It does not continually loop thru and check for past downloads as it doesn't track CDH downloads like that. Having a toggle to enable CDH if sab isn't enabled wouldn't matter as it's only tracking the active downloads so the button wouldn't do anything. 

Mylar will do manual post-processing against the download folder regardless of CDH being enabled or not. Which is probably the best that can be done for your specific use case.

And no, mylar doesn't touch the history of SAB. It just reads it to find the match against the current download.

**Comment by vp-en** [comment]
> Mylar will do manual post-processing against the download folder regardless of CDH being enabled or not. Which is probably the best that can be done for your specific use case outside of changing how CDH works.

Fair enough! I'll just do that for the time being. Still hoping for such a feature in the future I guess <3.

> And no, mylar doesn't touch the history of SAB. It just reads it to find the match against the current download.

Is there any way to make Mylar3 remove completed/imported items from the SABnzbd history? That would be so sweet. Doing it manually can be a bit of a chore.
